packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3845,7 +3802,7 @@ export class ContainerRuntime
 		};
 	}
 
-	private async closeStaleSummarizer(codePath: string): Promise<void> {
```

Comment:
was `codePath` param never used in here?

Reply:
Yep, it wasn't used.